sqlmesh/core/dialect.py

```diff
@@ -653,36 +654,46 @@ def extend_sqlglot() -> None:
 
 
 def select_from_values(
-    values: t.Iterable[t.Tuple[t.Any, ...]],
+    values: t.List[t.Tuple[t.Any, ...]],
```

Comment:
why did you have to change this? why not keep it an iterable?

Reply:
I need to be able to get the length of it